numba/targets/array.py

```diff
@@ -0,0 +1,32 @@
+from numba.array import Array
```

Comment:
There's already a `numba.targets.arrayobj`, so renaming this module to something more descriptive would be good :)